packages/prosess-vedtak/src/components/PreviewLink.jsx

```diff
@@ -0,0 +1,22 @@
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './vedtakForm.less';
+
+const PreviewLink = ({ previewCallback }) => (
+  <a
+    href=""
+    onClick={previewCallback}
+    onKeyDown={e => (e.keyCode === 13 ? previewCallback(e) : null)}
+    className={classNames(styles.previewLink, 'lenke lenke--frittstaende')}
+  >
+    <FormattedMessage id="VedtakForm.AutomatiskBrev.Lenke" />
+  </a>
+);
+
+PreviewLink.propTypes = {
+  previewCallback: PropTypes.func.isRequired,
+};
+
+export default PreviewLink;
```

Comment:
Ser at denne komponenten finnes mange andre steder også. Trenger ikke oppdatere alle stedene, men hvis du tar inn innholdet som en children-prop, så blir det enklere for de andre stedene å bytte ut, så man slipper å ha samme kopierte kodesnutt mange steder.  F eks noe sånt
```suggestion
const PreviewLink = ({ previewCallback, children }) => (
  <a
    href=""
    onClick={previewCallback}
    onKeyDown={e => (e.keyCode === 13 ? previewCallback(e) : null)}
    className={classNames(styles.previewLink, 'lenke lenke--frittstaende')}
  >
    {children}
  </a>
);

PreviewLink.propTypes = {
  previewCallback: PropTypes.func.isRequired,
  children: PropTypes.node.isRequired,
};

export default PreviewLink;
```